lib/webview.dart

```diff
@@ -62,6 +65,26 @@ class _ScreenerAppState extends State<ScreenerApp> {
     };
   }
 
+  Future<void> _handleSignOut() async {
+    if (googleUser.length > 1) {
+      await _googleSignIn.disconnect();
+    }
+  }
+
+  Future<void> _handleSignIn() async {
+    try {
+      var user = await _googleSignIn.signIn();
+      if (user != null) {
+        var authToken = await user.authentication;
+        var accessToken = authToken.accessToken;
+        postFunction(googleLoginUrl, jsonEncode({'access_token': accessToken}),
+            requestMethod);
+      }
+    } catch (error) {
```

Comment:
Handle the exception.

Reply:
What should we do in this case? show an alert box? or just pass the post request and let the function give the error?

Reply:
Not too sure what to do here. 

We can display a "failed to login" or a generic message maybe but not sure if it's required.